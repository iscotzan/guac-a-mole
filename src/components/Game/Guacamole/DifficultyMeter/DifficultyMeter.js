import React from "react";
import { Circle } from "rc-progress";

  const DifficultyMeter = (props) =>{
    return (
      <div className="difficultyLvlContainer">
        <div className={"difficultyLvlText"}>
          lvl {props.difficultyLvl}
        </div>
        <Circle
          percent={props.difficultyPercent}
          strokeWidth="6"
          strokeLinecap="round"
          className={"difficultyCircle"}
          strokeColor={"#db2828"}
        />
      </div>
    );
  }


export default DifficultyMeter;
