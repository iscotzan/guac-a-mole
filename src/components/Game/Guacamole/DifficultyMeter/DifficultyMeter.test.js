import React from "react";
import DifficultyMeter from "./DifficultyMeter";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

const wrapper = shallow(<DifficultyMeter difficultyLvl={3} difficultyPercent={50} />);
describe("<DifficultyMeter />", () => {
  it("should be defined", () => {
    expect(wrapper).toBeDefined();
  });

  it("renders without crashing", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
