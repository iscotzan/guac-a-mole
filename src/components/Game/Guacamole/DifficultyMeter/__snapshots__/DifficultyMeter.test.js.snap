// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DifficultyMeter /> renders without crashing 1`] = `
<div
  className="difficultyLvlContainer"
>
  <div
    className="difficultyLvlText"
  >
    lvl 
    3
  </div>
  <Progress
    className="difficultyCircle"
    gapPosition="top"
    percent={50}
    prefixCls="rc-progress"
    strokeColor="#db2828"
    strokeLinecap="round"
    strokeWidth="6"
    style={Object {}}
    trailColor="#D9D9D9"
    trailWidth={1}
  />
</div>
`;
